lib/codecs/src/decoding/format/bytes.rs

```diff
@@ -25,6 +25,11 @@ impl BytesDeserializerConfig {
         BytesDeserializer::new()
     }
 
+    /// The data type of returned events
```

Comment:
Just one tiny nitpit: These comments usually end with a `.` as they would end up as sentences in the HTML doc (if we configured that)

Reply:
Makes sense, I'll do that in a follow up.